src/models/seasonalStats.ts

```diff
@@ -0,0 +1,72 @@
+import mongoose from 'mongoose';
+
+// SCHEMA SETUP
+const seasonalStatsSchema = new mongoose.Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'user',
+    required: true,
+  },
+  season: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'season',
+    required: true,
+  },
+
+  rating: {
+    type: Number,
+    default: 1500, // TODO-kev: Consider if there is an enum to share here
+    min: [0, 'Must be greater than or equal to 0, but got {VALUE}.'],
+  },
+  highestRating: {
+    type: Number,
+    default: 0,
+    min: [0, 'Must be greater than or equal to 0, but got {VALUE}.'],
+  },
+  ratingBracket: {
+    type: String,
+    default: 'unranked', // TODO-kev: Consider if there is an enum to share here. Or if this is needed
+    enum: [
+      'unranked',
+      'iron',
+      'bronze',
+      'silver',
+      'gold',
+      'platinum',
+      'diamond',
+      'champion',
+    ],
+  },
```

Comment:
If you also decide to put the mappings in the db instead of the code constants, let's also make sure the server only pulls that mapping once on startup rather than on every game finish.